components/VotesList/VotesListItem.tsx

```diff
@@ -191,15 +199,42 @@ export function VotesListItem({
     return "transparent";
   }
 
-  function getTitleMaxWidth() {
-    if (activityStatus === "upcoming") return "70vw";
-    if (activityStatus === "active" && phase === "commit")
-      return "max(35vw, 320px)";
-    if (
-      (activityStatus === "active" && phase === "reveal") ||
-      activityStatus === "past"
-    )
-      return "max(400px, 45vw)";
+  function getCellWidths() {
```

Comment:
we now use different table cell widths depending on the activity status and phase. (used to only change the title's width)